components/TextFields/src/ContainedInputView/private/MDCTextControlVerticalPositioningReferenceBase.m

```diff
@@ -15,89 +15,187 @@
 #import "MDCTextControlVerticalPositioningReferenceBase.h"
 #import "MaterialMath.h"
 
-static const CGFloat kPaddingBetweenTopAndFloatingLabel = (CGFloat)10.0;
-static const CGFloat kPaddingBetweenFloatingLabelAndText = (CGFloat)6.0;
-static const CGFloat kPaddingBetweenTextAndBottom = (CGFloat)10.0;
-static const CGFloat kPaddingAboveAssistiveLabels = (CGFloat)0.0;
-static const CGFloat kPaddingBelowAssistiveLabels = (CGFloat)6.0;
+static const CGFloat kMinPaddingBetweenContainerTopAndFloatingLabel = (CGFloat)6.0;
```

Comment:
Since the base style is supposed to be the absence of a style these values don't really come from anywhere. I'll add a comment like that for the other styles though!